src/test/jtx/Env.h

```diff
@@ -147,6 +149,11 @@ class Env
         memoize(Account::master);
         Pathfinder::initPathTable();
         construct(std::forward<Args>(args)...);
```

Comment:
Why not simply: 

    // Enable the invariant enforcement amendment by default.
    construct(features(featureEnforceInvariants), std::forward<Args>(args)...);

As for disabling it, two questions:

1. Under what circumstances would we not want to have this? If the answer is "no reason" then we don't even need to mention this in a comment.
2. The technique you document on how to disable the invariant checking isn't what you, yourself, actually use: instead you use the `ENFORCE_INVARIANTS` configuration option. I think using that makes more sense.